source/system/Buffer.ooc

```diff
@@ -49,7 +49,7 @@ Buffer: cover {
 			start = this _size + start
 		if (distance < 0)
 			start -= size
-		start < this size ? This new(this _pointer + start, size minimum(this _size - start)) : This empty
+		start < this size ? This new(this _pointer as Byte* + start, size minimum(this _size - start)) : This empty
```

Comment:
These are the kinds of bugs you discover when writing tests. Pointer arithmetic, appearently, do not work with `Pointer`s...?